cloud-sql/mysql/tink/main.py

```diff
@@ -0,0 +1,237 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import datetime
+import logging
+import os
+import sqlalchemy
+import sys
+
+import tink
+from tink import aead
+from tink.integration import gcpkms
+
+
+# [START cloud_sql_mysql_cse_key]
+def init_tink():
+    aead.register()
+    key_uri = "gcp-kms://" + os.environ["KMS_KEY_URI"]
+    credentials = os.environ.get("GOOGLE_APPLICATION_CREDENTIALS", "")
+
+    try:
+        gcp_client = gcpkms.GcpKmsClient(key_uri, credentials)
+        gcp_aead = gcp_client.get_aead(key_uri)
+    except tink.TinkError as e:
+        logging.error('Error initializing GCP client: %s', e)
+        return None
```

Comment:
We should probably just throw this error rather than have get a None type error later on

Reply:
Done